jujugui/static/gui/src/app/models/models.js

```diff
@@ -2713,16 +2713,16 @@ YUI.add('juju-models', function(Y) {
     _collapseMachineConstraints: function(constraints) {
       var constraint = '';
       var constraintMap = {
-        arch: 'arch',
+        availabilityZone: 'availability-zone',
         cpuCores: 'cpu-cores',
         cpuPower: 'cpu-power',
-        mem: 'mem',
         disk: 'root-disk'
```

Comment:
@bac the `constraintMap` is a map to fix the constraints which use a different format than Juju expects, ex ) cpuCores > cpu-cores